zeebe/exporters/elasticsearch-exporter/src/test/java/io/camunda/zeebe/exporter/ElasticsearchExporterIT.java

```diff
@@ -195,4 +210,165 @@ final var record = factory.generateRecord();
         .extracting(ComponentTemplateWrapper::name)
         .isEqualTo(config.index.prefix);
   }
+
+  private boolean export(final Record<?> record) {
+    exporter.export(record);
+    exportedRecordsCounter++;
+    return true;
+  }
+
+  @Nested
+  @TestMethodOrder(OrderAnnotation.class)
+  final class IndexSettingsTest {
+    @Test
+    @Order(1)
```

Comment:
❓ why does the order matter?

Reply:
will remove now, it mattered when I relied on state (which I hated, it's ugly)